src/lib/Microsoft.Health.Expressions/Microsoft.Health.Expressions.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFramework>netstandard2.0</TargetFramework>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
This is technically a breaking change.  I assume this is needed because of the addition of JmesPath.Net?  This is currently used in the RP for validation.  I believe netcore3.1 will be compatible there but can you add a work item to update and verify after this PR is complete?

Reply:
This is because I wanted to incorporate `Microsoft.Health.Logging` which is `netcoreapp3.1`

Reply:
Actually it appears that the project in the RP that is using `Microsoft.Health.Fhir.Ingest.Template` has a targetFramework of `net472`, which only supports `netstandard2.0`. So I don't think that this will work. I'm assuming that the RP needs .netframework and runs on Windows so I'll have to adjust things here.

Reply:
Sorry yes, the template will be used in the RP code.  It should be fine but we should verify.

Reply:
I just built a local `Microsoft.Health.Fhir.Ingest.Template` nuget and added it to the  RP project. It doesn't work; apparently netcoreapp3.1 supports netstandard2.1 and `net472` only supports `netstandard2.0`. I've reverted this back to `netstandard2.0`. I'm also going to swtich `Microsoft.Health.Logging` to `netstandard2.0` as it doesn't need to be set to `netcoreapp3.1`

Reply:
Actually, I just tested locally and it doesn't seem like there is any reason why the logging library can't be netstandard2.0.  Can we update it and change these back to netstandard2.0?

Reply:
Yup, I've done just that and pushed up the changes.